assets/xml/objects/object_bal.xml

```diff
@@ -1,81 +1,93 @@
+        <Texture Name="gTingleMapTLUT" OutName="tingle_map_tlut" Format="rgba16" Width="4" Height="4" Offset="0x53B0" />
+        <Texture Name="gTingleBriefsTex" OutName="tingle_briefs" Format="rgba16" Width="16" Height="16" Offset="0x53D0" />
+        <Texture Name="gTingleTopFrontTex" OutName="tingle_top" Format="rgba16" Width="32" Height="32" Offset="0x55D0" />
+        <Texture Name="gTingleBalloonTex" OutName="tingle_balloon" Format="rgba16" Width="16" Height="16" Offset="0x5DD0" />
+        <Texture Name="gTingleBackpackTex" OutName="tingle_backpack" Format="rgba16" Width="8" Height="8" Offset="0x5FD0" />
+        <Texture Name="gTingleEyeOpenTex" OutName="tingle_eye_open" Format="rgba16" Width="32" Height="64" Offset="0x6050" />
+        <Texture Name="gTingleBodysuitTex" OutName="tingle_bodysuit" Format="rgba16" Width="8" Height="8" Offset="0x7050" />
+        <Texture Name="gTingleNostrilTex" OutName="tingle_nostril" Format="rgba16" Width="16" Height="16" Offset="0x70D0" />
+        <Texture Name="gTingleMouthTex" OutName="tingle_mouth" Format="rgba16" Width="64" Height="32" Offset="0x72D0" />
+        <Texture Name="gTingleNoseSideTex" OutName="tingle_nose" Format="rgba16" Width="16" Height="32" Offset="0x82D0" />
+        <Texture Name="gTingleHandTex" OutName="tingle_hand" Format="rgba16" Width="32" Height="16" Offset="0x86D0" />
+        <Texture Name="gTinglePenTopTex" OutName="tingle_pen_top" Format="rgba16" Width="16" Height="16" Offset="0x8AD0" />
+        <Texture Name="gTingleMapTex" OutName="tingle_map" Format="ci4" Width="64" Height="64" Offset="0x8CD0" />
+        <Texture Name="gTingleEyeClosedTex" OutName="tingle_eye_closed" Format="rgba16" Width="32" Height="64" Offset="0x94D0" />
+
+    <!-- Skeleton -->
+        <Limb Name="gTingleRootLimb" Type="Standard" EnumName="TINGLE_LIMB_ROOT" Offset="0xA4D0" />
+        <Limb Name="gTinglePelvisLimb" Type="Standard" EnumName="TINGLE_LIMB_PELVIS" Offset="0xA4DC" />
+        <Limb Name="gTingleBodyLimb" Type="Standard" EnumName="TINGLE_LIMB_BODY" Offset="0xA4E8" />
+        <Limb Name="gTingleUpperBodyLimb" Type="Standard" EnumName="TINGLE_LIMB_UPPER_BODY" Offset="0xA4F4" />
+        <Limb Name="gTingleBackRootLimb" Type="Standard" EnumName="TINGLE_LIMB_BACK_ROOT" Offset="0xA500" />
+        <Limb Name="gTingleBalloonLimb" Type="Standard" EnumName="TINGLE_LIMB_BALLOON" Offset="0xA50C" />
+        <Limb Name="gTingleRightArmRootLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_ARM_ROOT" Offset="0xA518" />
+        <Limb Name="gTingleRightShoulderLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_SHOULDER" Offset="0xA524" />
+        <Limb Name="gTingleRightUpperArmLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_UPPER_ARM" Offset="0xA530" />
+        <Limb Name="gTingleRightForearmLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_FOREARM" Offset="0xA53C" />
+        <Limb Name="gTingleRightWristLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_WRIST" Offset="0xA548" />
```

Comment:
For this one, the codebase itself isn't super consistent right now. Some objects call this something like `RightHandRootLimb`, others call it `RightWrist` like you did here. I prefer the former, because if you look at the actual limbs, you can see that this limb contains an offset, then the following limb actually draws something; I feel that calling the former a "root limb" portrays that relationship better:
```c
StandardLimb gTingleRightWristLimb = { 
    { 353, 0, 0 }, TINGLE_LIMB_RIGHT_HAND - 1, LIMB_DONE,
    NULL
};

StandardLimb gTingleRightHandLimb = { 
    { 0, 0, 0 }, TINGLE_LIMB_HELD_PEN - 1, LIMB_DONE,
    gTingleRightHandDL
};
```
However, like I said, we're currently quite inconsistent, so don't take this as a requirement or anything.